monitoring/Dockerfile-indexer

```diff
@@ -0,0 +1,31 @@
+FROM alpine:latest
+
+RUN apk add --no-cache git make musl-dev go bash libtool autoconf automake build-base libsodium-dev boost-dev
+
+# Configure Go
+ENV GOROOT /usr/lib/go
+ENV GOPATH /go
+ENV PATH /go/bin:$PATH
+
+RUN mkdir -p ${GOPATH}/src ${GOPATH}/bin
+
+# Build and run indexer
+RUN mkdir /work
+WORKDIR /work
+ADD . ./
+RUN make
+WORKDIR /work/cmd/algorand-indexer
+ENV CGO_ENABLED="1"
+RUN go build
+
+# The sleep is to wait until postgres starts
+CMD ["/bin/sh", "-c", "\
+  echo $ALGOD_NET && \
+  echo $CONNECTION_STRING &&\
+  sleep 5 && \
+  ./algorand-indexer daemon \
+    --server \":${PORT}\" \
+    -P \"${CONNECTION_STRING}\" \
+    --metrics-mode VERBOSE \
+    --algod-net \"${ALGOD_NET}\" \
+    --algod-token ${ALGOD_TOKEN}"]
```

Comment:
I wasn't able to get alpine working, @bricerisingalgorand suggested there is an issue with musl and I should try ubuntu. This configuration started:
```
ARG GO_VERSION=1.17.5
FROM golang:$GO_VERSION

ENV DEBIAN_FRONTEND noninteractive
RUN apt-get update && apt-get install -y apt-utils curl git git-core bsdmainutils python3


# Build go-algorand
RUN mkdir /work
WORKDIR /work
ADD . ./
WORKDIR /work/third_party/go-algorand
RUN ./scripts/configure_dev.sh
RUN make

# Build indexer
WORKDIR /work
RUN make
WORKDIR /work/cmd/algorand-indexer
ENV CGO_ENABLED="1"
RUN go build

# The sleep is to wait until postgres starts
CMD ["/bin/sh", "-c", "\
  echo $ALGOD_NET && \
  echo $CONNECTION_STRING &&\
  sleep 5 && \
  ./algorand-indexer daemon \
    --server \":${PORT}\" \
    -P \"${CONNECTION_STRING}\" \
    --metrics-mode VERBOSE \
    --algod-net \"${ALGOD_NET}\" \
    --algod-token ${ALGOD_TOKEN}"]
```